samples/subsys/display/lvgl/README.rst

```diff
@@ -61,6 +68,10 @@ or
 - :ref:`mimxrt1060_evk`
 - `RK043FN02H-CT`_
 
+Memory requirements
+===================
+The sample applications requires a LVGL memory pool of at least 32kBytes.
```

Comment:
```suggestion
The sample application requires an LVGL memory pool of at least 32 kBytes.
```

Reply:
Fixed in https://github.com/zephyrproject-rtos/zephyr/commit/800ff3dbbc54d3249f04a3522125877da76cbb6f#diff-a5534d3472f36eaab1e54778a5e0a130d17d06c6a4b83ca008da78a328e0f877